Production/python/sampleConfig.py

```diff
@@ -7,8 +7,9 @@
 from Configuration.Eras.Era_Run2_2016_cff import Run2_2016
 from Configuration.Eras.Era_Run2_2017_cff import Run2_2017
 from Configuration.Eras.Era_Run2_2018_cff import Run2_2018
+from Configuration.Eras.Era_Phase2C9_cff import Phase2C9
```

Comment:
will this work with 10_6_X? If not need to implement some workaround. For example use `if else` inside GetPeriodCfg and do an `import` only inside the corresponding `if` branch.

Reply:
I'll add a cmssw version specific `if else` then

Reply:
Done